linode/docs
# Fix aliases

- This adds a preceding forward slash to all aliases in the repo

- The change is compatible with both Hugo .52 and newer versions of Hugo

- The older relatively-style aliases were not working with newer versions of Hugo

- Some aliases already had a preceding forward slash, so those were not changed

- Commit 23eef3d was the result of running this script: [fix-aliases.py](https://gist.github.com/nmelehan/cff626c23a46c463f469b660235a8d2a). If you want to run this, put it at the root of your docs repo and run `./fix-aliases.py`

---

To test in the new UI:

1. `git fetch nmelehan && git checkout --track nmelehan/fix-aliases`

1. `git checkout feature/new-docs-ui`

1. `git checkout -b test-aliases-branch`

1. `git merge fix-aliases`

1. There will be a merge conflict in `_vendor/github.com/linode/linode-website-partials/header.html`. You don't really need to resolve this to test the aliases, but you will see some temporary visual garbage in the browser that can be ignored:

    <img width="1453" alt="Screen Shot 2020-09-15 at 5 46 08 PM" src="https://user-images.githubusercontent.com/255491/93268582-6540b900-f77b-11ea-94e4-2a37d7b84e46.png">

1. `hugo75 server`
